flang/tools/bbc/bbc.cpp

```diff
@@ -142,6 +142,12 @@ static llvm::cl::opt<bool>
                        llvm::cl::desc("enable openmp device compilation"),
                        llvm::cl::init(false));
 
+static llvm::cl::opt<std::string> enableDoConcurrentToOpenMPConversion(
+    "fdo-concurrent-parallel",
```

Comment:
Same here, the option name should probably have amp or smth that inform the user that this is targeting openmp under the hood.

Reply:
Done.